src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -648,6 +648,11 @@ InferenceProfiler::ProfileHelper(
   all_timestamps_.clear();
   previous_window_end_ns_ = 0;
 
+  // Start with a fresh empty timestamp vector in the manager
```

Comment:
This fixes the bug between rates around timestamps.  This function is called at the start of a new rate. This function clears its local vector of timestamps on line 648, but the manager is the one with the real data, and that one could have results from the end of the previous rate (mostly from "end of sequence" requests).  This forces the manager to clear its timestamps for the new rate